prisma-fmt/src/get_config.rs

```
@@ -1,86 +1,93 @@
-use psl::{parser_database::Files, Diagnostics};
-use serde::Deserialize;
-use serde_json::json;
+use psl::{diagnostics::DatamodelError, parse_configuration_multi_file, parser_database::Files};
+use serde::{Deserialize, Serialize};
 use std::collections::HashMap;
 
-use crate::{schema_file_input::SchemaFileInput, validate::SCHEMA_PARSER_ERROR_CODE};
+use crate::schema_file_input::SchemaFileInput;
 
 #[derive(Deserialize, Debug)]
 #[serde(rename_all = "camelCase")]
 struct GetConfigParams {
     prisma_schema: SchemaFileInput,
     #[serde(default)]
-    ignore_env_var_errors: bool,
```

Not needed anymore, consumers can choose to ignore the errors

---

Take that back, client will have a hard time without it, restoring